readthedocs/doc_builder/config.py

```diff
@@ -0,0 +1,114 @@
+
+from readthedocs_build.config import (ConfigError, BuildConfig,
+                                      load as load_config)
+
+
+class ConfigWrapper(object):
```

Comment:
Currently rtd-build just does validation on the YAML, and doesn't provide any accessor methods, just presenting the YAML object. We could easily add these methods, but in general the logic around which python interpreter we're calling on the command line is specific to RTD in some ways (eg. we call `python` instead of `python2`, but explicitly use `python3`, because of Ubuntu).